misc/documentation_styleguide.md

```diff
@@ -65,11 +66,24 @@ First level is flushed all the way to the left. A sub level is offset by 4 space
 
 ## Links
 
-When cross reference a document, there are three kinds: within a document, within a project group, and outside a project group. The `[displayed text]` should be to the point and within a pair of square brackets. The displayed text must be followed by `(the link)` enclosed in a pair parenthesis. Follow the formats below to ensure link connection.
+There are three kind of links when cross referencing a document: within a document, within a project group, and outside a project group. The `[displayed text]` should be to the point and within a pair of square brackets. The displayed text must be followed by `(the link)` enclosed in a pair of parenthesis. Follow these formats for each case:
 
-* Reference within the same document: the link is proceeded with a # - [General rules]`(#General rules)`.
-* Reference within a project group: the link is proceeded with ../ - [Firefox Focus]`(../products/focus/testing_focus.md)`.
-* Reference outside mozilla-l10n group: the link is a full path - [OpenDesign github repository]`(https://github.com/mozilla/OpenDesign/tree/master/2017)`.
+* Reference within the same document: link directly to the anchor, e.g. `[General rules](#general-rules)`. Note that GitHub automatically creates anchors for titles: if the title is *Title Example*, the anchor will be lowercase, with spaces replaced by dashes, i.e. `#title-example`.
+* Reference within a project group: use relative links, instead of absolute links (starting with `/`) or full GitHub URLs. For example, to link to a document in the parent folder, use `[other document](../other.md)`.
+* Reference outside the current repository: use the full GitHub URL, e.g. `[OpenDesign GitHub repository](https://github.com/mozilla/OpenDesign/tree/master/2017)`.
+
+## Images
+
+When adding images to a repository, make sure that the size is not too big for the content displayed on GitHub. In case of PNG files, make also sure to optimize files through online services like [TinyPNG](https://tinypng.com/).
+
+* Create a folder that maps to your folder. If there are too many images, create a subfolder for a particular document.
```

Comment:
Create a folder that maps to the main folder in which your documented is stored.